src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -860,6 +864,55 @@ private static void VerifyRefAssemblyClient(string lib_cs, string source, Action
         }
 
         [Theory]
+        [InlineData("")]
+        [InlineData(@"[assembly: System.Reflection.AssemblyVersion(""1"")]")]
+        [InlineData(@"[assembly: System.Reflection.AssemblyVersion(""1.0.*"")]")]
```

Comment:
See [the docs](https://msdn.microsoft.com/en-us/library/system.reflection.assemblyversionattribute(v=vs.110).aspx):

> You can specify all the values or you can accept the default build number, revision number, or both by using an asterisk (*). [...] The default build number increments daily. The default revision number is the number of seconds since midnight local time (without taking into account time zone adjustments for daylight saving time), divided by 2.

That is nondeterministic.